app/domain/operations/hbx_enrollments/drop_enrollment_members.rb

```diff
@@ -52,10 +55,45 @@ def drop_enrollment_members(params)
         Success(dropped_member_info)
       end
 
+      def remove_dropped_enr_members(new_enrollment, non_eligible_members)
+        ineligible_applicant_ids = non_eligible_members.pluck(:applicant_id)
+        new_enrollment.hbx_enrollment_members.where(:applicant_id.in => ineligible_applicant_ids).delete
+      end
+
+      def terminate_base_enrollment(base_enrollment, new_enrollment, terminate_date)
+        return if  base_enrollment.coverage_expired?
+
+        if base_enrollment.may_terminate_coverage? && (new_enrollment.effective_on > base_enrollment.effective_on)
+          base_enrollment.terminate_coverage!
+          base_enrollment.update_attributes!(terminated_on: terminate_date)
+        elsif base_enrollment.may_cancel_coverage?
+          base_enrollment.cancel_coverage!
+        end
+      end
+
+      def ivl_params(base_enrollment, new_effective_date)
+        { product_id: product_id_csr_variant(base_enrollment, new_effective_date.year),
+          consumer_role_id: base_enrollment.consumer_role_id,
```

Comment:
consumer_role_id: base_enrollment.consumer_role_id, not needed
Please also change method name to custom_params or something

Reply:
Since there would be different behavior for shop enrollment. Have been using the same name as being used in reinstatement.

Reply:
Please ignore reinstatement logic, make changes as this tool will be working for only IVL market for now.

Reply:
Removing consumer_role_id as suggested and updating method name.